requirements.txt

```diff
@@ -3,4 +3,5 @@ urllib3==2.2.1
 python-dotenv==1.0.1
 requests==2.31.0
 discord.py==2.3.2
-vectara-agentic==0.1.6
\ No newline at end of file
+vectara-agentic==0.1.10
```

Comment:
Maybe upgrade to 0.1.11?

Reply:
Latest version is throwing error about a module. will merge with that and later we can update.